java/dataset/src/test/java/org/apache/arrow/dataset/ParquetWriteSupport.java

```diff
@@ -42,13 +43,15 @@
   private final Schema avroSchema;
   private final List<GenericRecord> writtenRecords = new ArrayList<>();
   private final GenericRecordListBuilder recordListBuilder = new GenericRecordListBuilder();
+  private final Random random = new Random();
```

Comment:
Do we need to provide a seed for the object?

Reply:
Can you share some thoughts doing that? Thanks.

Reply:
I am not sure if it is a problem:
If we do not use a seed, the file name for each run is identical. So the result of the previous run may be overwritten by the next run.

Reply:
> If we do not use a seed, the file name for each run is identical

I am not sure I get you point here correctly, but did you mean "If we do not use a seed, the file name for each run is **not** identical"?

Reply:
I mean if we do not use a seed, it amounts to always setting 0 as the seed. So each time we run the program, we get the same random numbers (they are just pseudo-random), and the same file names.

Reply:
> it amounts to always setting 0 as the seed

I don't think it's normal to have a fixed seed in Random by default. Are you using a regular version of JVM?

Reply:
After investigating the source code, I find you are right. The default constructor uses a seed based on the current time.